pvr.mediaportal.tvserver/addon.xml.in

```diff
@@ -154,7 +154,7 @@
     <disclaimer lang="ms_MY">Ini merupakan perisian tidak stabil! Pengarang tidak bertanggungjawab atas kegagalan rakaman, pemasa tidak betul, masa yang dibazirkan, atau apa jua kesan yang tidak dikehendaki.</disclaimer>
     <disclaimer lang="mt_MT">Dan il-programm mhuwiex stabbli! L-Awturi m'humiex responsabbli bl-ebda mod għal rekordings li ma jirnexxewx, arloġġi żbaljati, siegħat moħlija, jew kwalunkwe effett ieħor mhux mixtieq.</disclaimer>
     <disclaimer lang="nb_NO">Dette er ustabilt programvare! Forfatterne er ikke ansvarlig på noen måte for ødelagte opptak, feilstilte tidsur, bortkastede timer, eller andre uønskede hendelser…</disclaimer>
-    <disclaimer lang="nl_NL">Deze software is niet 100% stabiel! De auteurs zijn op geen enkele wijze aansprakelijk voor mislukte opnames, verspilde tijd, of enig ander ongewild neveneffect.</disclaimer>
+    <disclaimer lang="nl_NL">Deze software is niet 100tabiel! De auteurs zijn op geen enkele wijze aansprakelijk voor mislukte opnames, verspilde tijd, of enig ander ongewild neveneffect.</disclaimer>
```

Comment:
Ping @gade01, there another one where the 100% was set :laughing:, can you look to make language update and request for here too?

Reply:
Seems was come from my update script, has created a request to correct by hand and will look later, what my script makes wrong.